index.js

```diff
@@ -162,12 +166,40 @@ function LandroidAccessory(platform, name, serial, accessory) {
     if(this.config.rainsensor && this.accessory.getService(Service.LeakSensor)) this.accessory.getService(Service.LeakSensor).on('get', this.getLeak.bind(this));
 }
 
-LandroidAccessory.prototype.landroidUpdate = function(mower, data) {
+LandroidAccessory.prototype.landroidUpdate = function(mower, data, mowdata) {
+  var totalTime, totalBladeTime, totalDistance;
+
   if(mower.raw.serial_number !== this.serial) return;
 
   if(data != null && data != undefined){
     let oldDataset = this.dataset;
     this.dataset = new LandroidDataset(data);
+    // this.log("landroidUpdate ran with RSSI " + this.dataset.wifiQuality + ", battery temperature " + this.dataset.batteryTemperature);
+		if(mowdata){
```

Comment:
Would be nice if this part would use two spaces as indentation like the rest of the code

Reply:
Ah OK. My editor uses width-2 tabs rather than spaces for indentation. I'll see if I can convert them somehow.

Reply:
Linux expand!